### PR TITLE
update virtual-env.ipynb documentation for new jupy

### PR DESCRIPTION
**Pull request by danykaufman**

### CODE REVIEW

File: virtual-env.ipynb [hunk 1]
```diff
@@ -26,78 +31,116 @@
...
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"preconfigured-conda-environments\"></a>\n",
+    "## Preconfigured Conda environments\n",
+    "\n",
+    "Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod:\n",
```
Comment (gilad-shaham) [inline virtual-env.ipynb]: Instead of Iguazio, we should say "the platform". Similarly in subsequent lines

File: virtual-env.ipynb [hunk 2]
```diff
@@ -26,78 +31,116 @@
...
+    "- **mlrun-base**: This environment includes MLRun and all its dependencies preinstalled.\n",
+    "- **mlrun-extended**: This environment includes all the packages from mlrun-base, as well as additional packages such as TensorFlow, PyTorch, and scikit-learn, which are required for the demo notebooks.\n",
+    "\n",
+    "While Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod, it's important to note that any Conda installations to these environments are not persistent. They will be reset when the Jupyter service is restarted.\n",
+    "\n",
+    "However, when these Conda environments are activated, the `PIP_PREFIX` and `PYTHONPATH` environment variables are automatically set to the data mount directory of the running user: `/User/.pythonlibs/<environment name>`. This means that PIP installations will persist even after restarting the Jupyter pod.\n",
+    "\n",
+    "Any new Conda environment that is created or cloned will be fully located in the data mount directory of the running user, specifically at `/User/.conda/<environment name>`. It is worth noting that these environments are fully persistent."
```
Comment (gilad-shaham) [inline virtual-env.ipynb]: > It is worth noting that these environments are fully persistent.
Without "It is worth nothing that"

**Review by gilad-shaham — CHANGES_REQUESTED** [review]

File: virtual-env.ipynb [hunk 3]
```diff
@@ -26,78 +31,116 @@
...
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"preconfigured-conda-environments\"></a>\n",
+    "## Preconfigured Conda environments\n",
+    "\n",
+    "Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod:\n",
+    "- **base**: This is the default base environment that includes the Conda binaries. It should not be used for development purposes.\n",
```
Comment (jillnogold) [inline virtual-env.ipynb]: Suggest replacing few instances of (unless there are circumstances in which you can, which should be specified):
It should not be used...
with:
Do not use it...

File: virtual-env.ipynb [hunk 4]
```diff
@@ -26,78 +31,116 @@
...
+    "- **jupyter**: This environment incudes the JupyterLab server and all of its dependencies. It should not be used for development purposes.\n",
+    "- **mlrun-base**: This environment includes MLRun and all its dependencies preinstalled.\n",
+    "- **mlrun-extended**: This environment includes all the packages from mlrun-base, as well as additional packages such as TensorFlow, PyTorch, and scikit-learn, which are required for the demo notebooks.\n",
+    "\n",
+    "While Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod, it's important to note that any Conda installations to these environments are not persistent. They will be reset when the Jupyter service is restarted.\n",
```
Comment (jillnogold) [inline virtual-env.ipynb]: While Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod, it's important to note that any Conda installations to these environments are not persistent. They are reset when the Jupyter service is restarted

File: virtual-env.ipynb [hunk 5]
```diff
@@ -26,78 +31,116 @@
...
+    "\n",
+    "However, when these Conda environments are activated, the `PIP_PREFIX` and `PYTHONPATH` environment variables are automatically set to the data mount directory of the running user: `/User/.pythonlibs/<environment name>`. This means that PIP installations will persist even after restarting the Jupyter pod.\n",
```
Comment (jillnogold) [inline virtual-env.ipynb]: "However, when these Conda environments are activated, the `PIP_PREFIX` and `PYTHONPATH` environment variables are automatically set to the data mount directory of the running user: `/User/.pythonlibs/<environment name>`. This means that PIP installations persist even after restarting the Jupyter pod.\n",

File: virtual-env.ipynb [hunk 6]
```diff
@@ -26,78 +31,116 @@
...
+    "   \n",
+    "   <br>For example, the following command activates an environment named \"myenv\":\n",
     "   ```sh\n",
-    "   conda install -n myenv scipy pandas=0.25.0 tensorflow=1.13.1 python=3.7\n",
-    "   ```\n",
+    "   conda activate myenv\n",
+    "   ```   \n",
+    "   <br>\n",
     "\n",
+    "4. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Once you activate your environment, you can use either PIP or Conda to install the necessary Python packages.<br> To do this, replace `<package>` with the name of the package you want to install, and optionally add `==<version>` for PIP or `=<version>` for Conda. You can specify multiple packages in the same command.<br>\n",
```
Comment (jillnogold) [inline virtual-env.ipynb]: "4. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Use either PIP or Conda to install the necessary Python packages.<br> To do this, replace `<package>` with the name of the package you want to install, and optionally add `==<version>` for PIP or `=<version>` for Conda. You can specify multiple packages in the same command.<br>\n",

File: virtual-env.ipynb [hunk 7]
```diff
@@ -26,78 +31,116 @@
...
+    "   Using PIP:\n",
+    "   ```sh\n",
+    "   pip install <package> [<package> ...]\n",
+    "   ```\n",
+    "   <br>Alternatively, using Conda:\n",
+    "   ```sh\n",
+    "   conda install <package> [<package> ...]\n",
+    "   ```\n",
+    "   \n",
+    "   <br>For example, the following command uses PIP to install the SciPy, pandas version 1.4.4, and TensorFlow version 2.9.3 packages for the \"myenv\" environment that we activated in the previous step:\n",
```
Comment (jillnogold) [inline virtual-env.ipynb]: "   <br>For example, the following command uses PIP to install the SciPy, pandas version 1.4.4, and TensorFlow version 2.9.3 packages for the \"myenv\" environment that you activated in the previous step:\n",
    "   ```sh\n",

File: virtual-env.ipynb [hunk 8]
```diff
@@ -26,78 +31,116 @@
...
+    "   <br>To shorten this command, you can use the `/User` data mount to the running-user directory (see [Platform Data Containers](data-ingestion-and-preparation/README.ipynb#platform-data-containers)):\n",
+    "   ```sh\n",
+    "   conda env -n myenv export > /User/virtual_env/myenv.yaml\n",
+    "   ```\n",
+    "   <br>\n",
     "\n",
-    "5.  <a id=\"conda-virt-env-setup-step-refresh-ui\"></a>Refresh the JupyterLab UI to apply your changes.\n",
-    "    After refreshing the UI, you should be able to see your new environment in the list of available kernels in Jupyter Notebook."
+    "6. <a id=\"conda-virt-env-setup-step-refresh-ui\"></a>Refresh the JupyterLab UI to apply your changes.<br>\n",
+    "   After refreshing the UI, you should be able to see your new environment in the list of available kernels in JupyterLab."
```
Comment (jillnogold) [inline virtual-env.ipynb]: " After refreshing the UI, you should see your new environment in the list of available kernels in the JupyterLab."

File: virtual-env.ipynb [hunk 9]
```diff
@@ -26,78 +31,116 @@
...
+    "   conda env export -n <environment name> > /v3io/<container name>[/<directory path>]/<environment name>.yaml\n",
+    "   ```\n",
+    "   <br>It is recommended that you save the environment to a virtual-environments directory in your running-user home directory (**/v3io/users/&lt;running user&gt;**).\n",
+    "   \n",
+    "   For example, the following command creates a **/v3io/users/&lt;running user&gt;/virtual_env/myenv.yaml** file:\n",
+    "   ```sh\n",
+    "   conda env export -n myenv > /v3io/users/$V3IO_USERNAME/virtual_env/myenv.yaml\n",
+    "   ```\n",
+    "   \n",
+    "   <br>To shorten this command, you can use the `/User` data mount to the running-user directory (see [Platform Data Containers](data-ingestion-and-preparation/README.ipynb#platform-data-containers)):\n",
```
Comment (jillnogold) [inline virtual-env.ipynb]: "   <br>To shorten this command, use the `/User` data mount to the running-user directory (see [Platform Data Containers](data-ingestion-and-preparation/README.ipynb#platform-data-containers)):\n",
    "   ```sh\n",

File: virtual-env.ipynb (hunk 6 above)
Comment (danykaufman) [reply virtual-env.ipynb]: I would still leave "Once you activate your environment" to emphasize it is a required step.

**Comment by jillnogold** [comment]
approved

**Review by gilad-shaham — APPROVED** [review]

**Pull request merged**